app/services/slack_notification_service.rb

```diff
@@ -56,4 +107,19 @@ def filter_merged_action
     matches = slack_bot.find_message pr.url
     slack_bot.add_merge_emoji matches
   end
+
+  def js_channels(pr)
```

Comment:
@GuilleLeopold  I separated the cannel target depending on the technology if the PR's language is javascript (As it was made for the icons), but the thing here is if we have a repo with react code but without "react" in its name, the pr will be displayed on "javascript-reviewers" instead of "react-reviewers" (For example "forecast-web"), so I am not sure if this is the best solution of if it's better to group all javascript repos in just one channel. WDYT?

Reply:
I prefer this, to split into several javascript channel as grouping all the js channel would mean that Node and React go together but are quite different

Reply:
Done @GuilleLeopold